controllers/device_controller.go

```diff
@@ -256,28 +239,26 @@ func (r *DeviceReconciler) reconcileUpdateDevice(ctx context.Context, d *devicev
 
 // Update the actual property value of the device on edge platform,
 // return the latest status and the names of the property that failed to update
-func (r *DeviceReconciler) reconcileDeviceProperties(d *devicev1alpha1.Device, deviceStatus *devicev1alpha1.DeviceStatus, log logr.Logger) (*devicev1alpha1.DeviceStatus, []string) {
+func (r *DeviceReconciler) reconcileDeviceProperties(d *devicev1alpha1.Device, deviceStatus *devicev1alpha1.DeviceStatus) (*devicev1alpha1.DeviceStatus, []string) {
 	newDeviceStatus := deviceStatus.DeepCopy()
 	// This list is used to hold the names of properties that failed to reconcile
 	var failedPropertyNames []string
 	// 2. reconciling the device properties' value
-	log.V(3).Info("reconciling the device properties' value")
+	klog.V(3).Info("reconciling the value of device properties")
 	for _, desiredProperty := range d.Spec.DeviceProperties {
 		if desiredProperty.DesiredValue == "" {
 			continue
 		}
 		propertyName := desiredProperty.Name
 		// 1.1. gets the actual property value of the current device from edge platform
-		log.V(4).Info("getting the actual property state", "property", propertyName)
+		klog.V(4).Infof("getting the actual value of property: %s", propertyName)
 		actualProperty, err := r.deviceCli.GetPropertyState(nil, propertyName, d, iotcli.GetOptions{})
 		if err != nil {
 			failedPropertyNames = append(failedPropertyNames, propertyName)
 			continue
 		}
-		log.V(4).Info("got the actual property state",
-			"property name", propertyName,
-			"property getURL", actualProperty.GetURL,
-			"property actual value", actualProperty.ActualValue)
+		klog.V(4).Infof("got the actual property state, {Name: %s, GetURL: %s, ActualValue: %s}",
+			propertyName, actualProperty.GetURL, actualProperty.ActualValue)
```

Comment:
Adding device name into log message may be better.

Reply:
The deviceName has been added.